Update formats.py

NP2241 opened the pull request:
changed the comments to match documentation as detailed in https://code.djangoproject.com/ticket/35095#comment:6

NP2241 committed:
Update formats.py

changed the comments to match documentation as detailed in https://code.djangoproject.com/ticket/35095#comment:6

e11bits commented on django/conf/locale/de_CH/formats.py:
```diff
@@ -27,7 +27,7 @@
 # these are the separators for non-monetary numbers. For monetary numbers,
 # the DECIMAL_SEPARATOR is a . (decimal point) and the THOUSAND_SEPARATOR is a
-# ' (single quote).
+# , (comma).
```
I don't think it is a `comma` but also a `nbsp` as for plain numbers. (see my comment in the ticket). So I would think it would be correct to remove " and the THOUSAND_SEPARATOR is a x (whatever)." from the comment.

nependyala replied:
I have added a new commit taking into account the latest changes in the thread

NP2241 committed:
Update formats.py

changes added as based on the following thread https://code.djangoproject.com/ticket/35095#comment:6, from comment 20
